cli/azd/internal/repository/infra/main.bicep

```diff
@@ -0,0 +1,25 @@
+targetScope = 'subscription'
```

Comment:
Did you intend to check this file and the corresponding `main.parameters.json` in here as well?  Or should they just be in resources/minimal?

Reply:
Interesting. I may have ran init here